config/sidekiq.yml

```diff
@@ -0,0 +1,14 @@
+verbose: true
+logfile: ./log/sidekiq.log
+development:
+  :concurrency: 2
+staging:
+  :concurrency: 2
+demo:
```

Comment:
sandbox & demo are Trase-specific and can be removed safely

Reply:
yep, I was peeking on Trase, thanks